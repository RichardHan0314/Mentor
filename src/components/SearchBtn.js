import React from 'react';
import styled from 'styled-components';

const SearchBtn = () => {
    return (
        <SearchWrap>
            <SearchBlock>
                <SearchCircle />
                <SearchRectangle />
            </SearchBlock>
        </SearchWrap>
    );
};

const SearchWrap = styled.div`
    position: relative;
    width: 40px;
    height: 30px;
`;

const SearchBlock = styled.div`
    position: relative;
    width: 20px;
    margin: 0 auto;
    padding: 10px 0;
    text-align: center;
`;

const SearchCircle = styled.div`
    width: 10px;
    height: 10px;
    border: 4px solid #f65261;
    border-radius: 50%;
`;

const SearchRectangle = styled.div`
    position: absolute;
    right: 15px;
    bottom: 10px;
    width: 12px;
    transform: rotate(140deg);
    border: 1.5px solid #f65261;
`;

export default SearchBtn;
