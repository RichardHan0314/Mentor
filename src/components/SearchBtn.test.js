import React from 'react';
import { render } from '@testing-library/react';
import SearchBtn from './SearchBtn';

describe('SearchBtn component', () => {
    it('Render SearchBtn snapshot', () => {
        const { asFragment } = render(<SearchBtn />);
        expect(asFragment(<SearchBtn />)).toMatchSnapshot();
    });
});
