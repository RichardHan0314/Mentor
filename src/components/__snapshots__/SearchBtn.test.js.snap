// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBtn component Render SearchBtn snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm dtWMrr"
  >
    <div
      class="sc-AxirZ jpTJgh"
    >
      <div
        class="sc-AxiKw hTVPjm"
      />
      <div
        class="sc-AxhCb daOEwe"
      />
    </div>
  </div>
</DocumentFragment>
`;
